deploy/helm/sumologic/conf/traces/traces.conf

```diff
@@ -0,0 +1,17 @@
+<source>
+  @type zipkin
+  @label @TRACING
+  port 9411
+</source>
+<label @TRACING>
+  <filter tracing.**>
+    @type stdout
+  </filter>
```

Comment:
By doing this, fluentd will log the traces to STDOUT right? I assume you are doing this for debugging but keep the challenge is we collect the fluentd logs in the K8s collection process and therefore customers will have an ingest spike for this.  If so, we should probably make this configurable and not on by default.